src/index.js

```diff
@@ -0,0 +1,30 @@
+const express = require("express")
+const path = require("path")
+const app = express()
+require('dotenv').config()
+const port = process.env.PORT || 3000;
+const fetch = require("node-fetch")
+
+app.use(express.json());
+app.use(express.static(path.join(__dirname, '..', 'public')))
+app.use(express.urlencoded({ extended: false }));
+app.post("/search", (req, res) => {
+    let name = req.body.q
+    let headers = { Accept: "application/json" }
+    let url = `https://api.adzuna.com/v1/api/jobs/gb/search/1?app_id=${process.env.ID}&app_key=${process.env.TOKEN}&what=${name}`
+    fetch(url, { method: 'GET', headers: headers })
```

Comment:
See! here the api you are calling is using GET and already using querystring for this endpoint meanwhile you converted it to POST!